### PR TITLE
Updating READMEs (issues 815 and 863)

### LINKED ISSUE #815: Update dashboard and main README
After the redesign discussion #791 , we should change the 

- [x] [dashboard README](https://github.com/dianna-ai/dianna/tree/main/dianna/dashboard#dianna-dashboard).

For example, the first paragraph can be changed to 

> The DIANNA dashboard can be used for explanation of the behaviour of several ONNX models trained for the tasks and datasets presented in the [DIANNA Tutorials](https://github.com/dianna-ai/dianna/tree/main/tutorials#datasets-and-tasks ). The dashboard shows the visual explanation of a models' decision on a selected data item by a selected XAI method (explainer). It allows you to compare the results of different explainers, as well as explanations of the top ranked predicted labels. The dashboard was created using [streamlit](https://streamlit.io/ ).

- Also, should change the dashboard screenshot in
- [x] the main README.
- [x] dashboard README

### EU law use-case

- [x] Add to datasets table in main README
~~- [ ] Update RSD pages with the dataset~~

### Demo tutorial [video](https://github.com/dianna-ai/dianna?tab=readme-ov-file#demo-movie-update-planned) using the dashboard:

- [x] Script
- [x] Video recording using streamlit
- [x] Uploadvideo to eScience center YouTube channel
- [x] Update RSD pages
- [x] Link to the video in main README and dashboard README

### LINKED ISSUE #863: Use collapsed sections in main and tutorial README
- Visually shorten the main and tutorial READMEs  using [collapsed sections](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-collapsed-sections): 

```
<details><summary>Topic to expand below</summary>
<p>
 Here the expanded explanations.
</details>

```
this is the result
> <details><summary>Topic to expand below</summary>
> <p>
>  Here the expanded explanations.
> </details>

- It the details are in a _table_, **do not use** `<p>`!

Apply this to 

### main README

- [x] [Pre-requisites only for Macbook Pro with M1 Pro chip users](https://github.com/dianna-ai/dianna#pre-requisites-only-for-macbook-pro-with-m1-pro-chip-users)
- [x] [Text and image usage examples](https://github.com/dianna-ai/dianna#text-and-image-usage-examples)
- [x] [Datasets](https://github.com/dianna-ai/dianna#datasets), **each of the modality**
- [x] [ONNX model](https://github.com/dianna-ai/dianna#datasets), **each of the modality**; rename to Model**s**

###  tutorials README

- [Datasets and Tasks](https://github.com/dianna-ai/dianna/tree/main/tutorials#datasets-and-tasks)
- [x] Illustratative
- [x] Simple
- [Summary of All Tutorials](https://github.com/dianna-ai/dianna/tree/main/tutorials#summary-of-all-tutorials)
- [x] Illustratative
- [x] Simple
- [x] [Hyperparameters](https://github.com/dianna-ai/dianna/tree/main/tutorials#important-hyperparameters)

### PR DESCRIPTION
**Pull request by elboyran**
Solves issues #815 and #863.

### CODE REVIEW

**Comment by elboyran** [comment]
Oh, as a Draft it passed all checks.... Looking at the fails now....

**Comment by loostrum** [comment]
> Oh, as a Draft it passed all checks.... Looking at the fails now....

That may be because the downloader test doesn't run on draft PRs. We disabled that to avoid many downloads while we don't change the downloader itself that much.

**Comment by elboyran** [comment]
@loostrum Today I saw you didn't review my PR yet, so I quickly grabbed the chance ;-)
 to edit a bit more the READMEs and while all checks were green yesterday, now (after Laura's PR?) the EU law notebook fails on Ubuntu. @cwmeijer any idea why?

**Comment by cwmeijer** [comment]
The error is:
"RuntimeError: operator torchvision::nms does not exist" while importing from torchvision.
However, all of the systems have the same torchvision version (0.18.1).
Pytorch-lightning versions are the same for all 3 as well. Same goes for transformers, which is the package bringing in torchvision as a dependency, all have version 4.46. No clue what is causing this.

**Comment by elboyran** [comment]
@loostrum would you have time to give it a go (review and hopefully error fix)?

**Comment by cwmeijer** [comment]
or @SarahAlidoost for that matter as she managed to fix a possibly similar issue a few weeks back. Whoever of you has time for this.

**Comment by loostrum** [comment]
I'm looking into it now. I do see that on ubuntu, the last passing run had an older version of transformers (4.45.2 vs 4.46.0). the 4.46.0 release of transformers was removed from pypi because of some issue with python 3.8, but that's not the version we're using. Just to be sure I've restarted the workflow, which is now using transformers 4.46.1. If that doesn't work, I'll try pinning to <4.46.0

**Comment by loostrum** [comment]
Pinning to <4.46 worked 🎉

File: dianna/dashboard/readme.md [hunk 1]
```diff
@@ -21,26 +19,8 @@ Open the link on which the app is running. Note that you are running the dashboa
...
-- Set the method specific settings to the selected XAI methods. If you don't change these, the default values are used.
-
-### Text
-
-- Input the text you want to use in the "Input string" box. Press enter to update the explanation.
-- Click on select model and select your trained model used for the prediction and explanation.
-- Click on select labels and select a text file containing the labels of your trained model. Labels should be separated by line breaks in the file, no other separators should be used. Make sure that both the <ins>labels</ins> and <ins>ordering of the labels</ins> is correct in correspondence to your trained model.
-- Check the XAI methods you want to use, multiple methods can be used at the same time for comparison.
-- Your image explanation will start loading.
+This [video](https://youtu.be/9VM5acip2s8) shows you how to use the DIANNA dashboard for some of the <img width="25" alt="Streamlit Logo" src="https://github.com/user-attachments/assets/2cac7d5d-c11a-48fe-b58e-71b15baaa163"> [tutorials](../../tutorials/README.md#summary-of-all-tutorials) accessible interactively in the input-output ends.
```
Comment (cwmeijer) [inline dianna/dashboard/readme.md]:
```suggestion
This [video](https://youtu.be/9VM5acip2s8) shows you how to use the DIANNA dashboard for some of the <img width="25" alt="Streamlit Logo" src="https://github.com/user-attachments/assets/2cac7d5d-c11a-48fe-b58e-71b15baaa163"> [tutorials](../../tutorials/README.md#summary-of-all-tutorials) to interactively explore the use cases from some of DIANNA's . 
```

**Review by cwmeijer — APPROVED** [review]
Great additions! Excellent idea to have these icons for the dashboard supported cases!

**Pull request merged**